recipes/glaze/all/conanfile.py

```diff
@@ -0,0 +1,77 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.files import get, copy
+from conan.tools.build import check_min_cppstd
+from conan.tools.scm import Version
+from conan.tools.layout import basic_layout
+import os
+
+required_conan_version = ">=1.50.0"
```

Comment:
```suggestion
required_conan_version = ">=1.51.1"
```

To ensure consumers have this fix https://github.com/conan-io/conan/pull/11790 for legacy generators.

Reply:
Thank you for your comment!